targets/TARGET_RENESAS/TARGET_RZ_A1XX/TARGET_VK_RZ_A1H/device/os_tick_ostm.c

```diff
@@ -191,5 +191,10 @@ uint32_t OS_Tick_GetOverflow (void)
   return (IRQ_GetPending(OSTM_IRQn));
 }
 
+// Get Cortex-A9 OS Timer interrupt number
+IRQn_ID_t mbed_get_a9_tick_irqn(){
```

Comment:
I am not certain why we dont use OS_Tick_GetIRQn but have this additional function?

Reply:
When in non-tickless mode, ARM-A targets need to provide all the `OS_Tick` functions, while mbed core provides it for ARM-M. It's the timer porting layer for RTX.

When in tickless mode, mbed_rtx_idle.c provides all the `OS_Tick` functions, built on the LPTicker SysTimer. And on ARM-A it provides that with the `mbed_get_a9_tick_irqn` as a building-block.

I guess just an API layering thing.

I didn't think anything had changed in that bit of the structure in this PR...